views/FanView/FanView.tsx

```diff
@@ -0,0 +1,104 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { FC } from 'react';
+import React, { View } from 'react-native';
+import {
+  Button,
+  Input,
+  Navbar,
+  Typography,
+  useInputValue,
+} from '../../.storybook/stories';
+import { selectDeviceWithId } from '../../redux/devicesSlice';
+import { useAppSelector } from '../../redux/hooks';
+import { Fan, RootStackParamList } from '../../types';
+import { sendAPIRequest } from '../../utils';
+import { styles } from './FanView.styles';
+
+type FanViewProps = NativeStackScreenProps<RootStackParamList, 'Fan'>;
+
+export const FanView: FC<FanViewProps> = ({ route }) => {
+  const { deviceId, nodeId, location } = route.params;
+
+  const fan = useAppSelector(state =>
+    selectDeviceWithId(state, location, nodeId, deviceId),
+  ) as Fan;
+
+  const [temp, onTempChange] = useInputValue(fan.values.V_TEMP);
+  const [direction, onDirectionChange] = useInputValue(fan.values.V_DIRECTION);
+  const [percantage, onPercentageChange] = useInputValue(
+    fan.values.V_PERCENTAGE,
+  );
+
+  const handleChangeFanParams = () => {
```

Comment:
Do we want to send 3 different messages instead of one?
e.g.
```
    sendAPIRequest({
      ...route.params,
      action: 'set',
      additionalParams: `V_TEMP=${temp}&V_DIRECTION=${direction}&V_PERCENTAGE=${percantage}`,
    });
```

Reply:
Yeah, because it's not supported on the backend site yet, I talked with @bchwast and he will handle such requests in the future. When it will be done, we can refactor it